numpy/__init__.pyi

```diff
@@ -143,6 +143,14 @@ class _flagsobj:
     def __getitem__(self, key: str) -> bool: ...
     def __setitem__(self, key: str, value: bool) -> None: ...
 
+_ArrayLikeInt = Union[
+    int,
```

Comment:
Does [`int`, `integer`] include `bool`?

Reply:
It does as `bool` is a subclass of `int`.

There is, as far as I'm aware, no convenient way of annotating a function which accepts integers but rejects booleans.